index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device0width">
+  <title></title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a>the offer</a></li>
+        <li><a>portfolio</a></li>
+        <li><a>about us</a></li>
+        <li><a>contact</a></li>
+      </ul>
+    </nav>
+    <section>
+      <h2>hello!</h2>
+      <p>
+        We are creative agency based in Warsaw, Poland <br>
+        Creating projects for clients all over the world
+      </p>
+    </section>
+    <section>
+      <h2>design!</h2>
+      <p>
+        Good design is our domain <br>
+        We work in multidisciplinary teams.
+      </p>
+    </section>
+    <section>
+      <h2>projects!</h2>
+      <p>
+        We always select varied projects, <br>
+        to step out of comfort zone.
+      </p>
+    </section>
+  </header>
+
+  <main>
+    <p>scroll down to find out more about our studio</p>
+    <article>
+      <h2>works</h2>
+      <article class="left-aside">
+        <article>
+          <h3>pineapp.</h3>
+          <p>
+            Period: March-April 2016 <br>
+            Team: John (UI), Edwin (UX) <br>
+            Jessica (Front-end)
+          </p>
+          <a href="">visit the website</a>
+        </article>
+        <aside>
+          <img src="pineapple.jpg">
```

Comment:
I's a good practice accessibility-wise, to add `alt=""`, but for decorative elements the alt should be left empty. I'd say it's a decorative image.

_Marking an image as decorative is quite easy. However, making an image decorative may also work against years of process on your part.
You must always supply an ALT attribute for an image, but that does not mean you have to provide a description. To mark an image as decorative, all that is required is that you supply an empty ALT attribute._

https://www.portent.com/blog/design-dev/decorative-images-how-to-identify-and-tag-them-for-accessibility.htm